lib/src/objects/start_options.dart

```diff
@@ -51,6 +52,13 @@ class StartOptions {
   /// option.
   final bool autoZoom;
 
+  /// If set this sets the zoom scale factor when the camera launches
```

Comment:
A suggestion for the entire doc comment:

```suggestion
  /// The initial zoom scale factor for the camera.
  ///
  /// Currently only supported on iOS and Android.
```